torch/_inductor/compile_fx.py

```diff
@@ -936,15 +936,17 @@ def clone_preserve_strides(x: torch.Tensor):
 
 
 def copy_misaligned_inputs(
-    new_inputs: List[torch.Tensor], check_inputs_idxs: Sequence[int]
+    new_inputs: List[Union[torch.Tensor, int]], check_inputs_idxs: Sequence[int]
 ) -> None:
     for i in check_inputs_idxs:
-        if new_inputs[i].data_ptr() % ALIGNMENT:
-            new_inputs[i] = clone_preserve_strides(new_inputs[i])
+        _inp = new_inputs[i]
+        assert isinstance(_inp, torch.Tensor)
```

Comment:
i dont actually think you need to assign to local to refine list element.. could be wrong though